core/azure-core-management/src/main/java/com/azure/core/management/PollStrategy.java

```diff
@@ -4,7 +4,7 @@
 package com.azure.core.management;
 
 import com.azure.core.exception.HttpRequestException;
-import com.azure.core.http.ContextData;
+import com.azure.core.Context;
```

Comment:
🏆

Reply:
Unlike Jianghao I'm not 100% convinced of this :-) I don't like polluting the root package needlessly, but I don't know of a better location either.

Reply:
Any opinions on com.azure.core.models.Context?

Reply:
Only negative ones....let's brainstorm tomorrow.